chia/types/peer_info.py

```diff
@@ -43,10 +46,12 @@ def port(self) -> uint16:
     # Functions related to peer bucketing in new/tried tables.
     def get_key(self) -> bytes:
         if self.ip.is_v4:
-            key = ipaddress.IPv6Address(int(ipaddress.IPv6Address("2002::")) | (int(self.ip) << 80)).packed
+            # key = ipaddress.IPv6Address(ipv6_2002_int | (int(self.ip) << 80)).packed
+            # below is a micro-optimization of above.  it avoids one creation of an ip address object.
+            key = (ipv6_2002_int | (int(self.ip) << 80)).to_bytes(16, byteorder="big")
         else:
             key = self.ip.packed
-        key += bytes([self.port // 0x100, self.port & 0x0FF])
+        key += bytes((self.port // 0x100, self.port & 0x0FF))
```

Comment:
doesn't this change the meaning? in an array of two ints the same as a tuple of two ints when passed to `bytes()`?